Rules.md

```diff
@@ -197,13 +195,13 @@ A battle has one or more rounds. Every round has following parts:
         * **5-6:** Kills an enemy unit (*)
 
     Kills are resolved before wounds.
-1. **Attacker decides to retreat or has no healthy units left:** All units move back to the last
-   touched unoccupied, sea or owned region.
+1. **Attacker decides to retreat or has no healthy units left:** All units move back to the region
+   where to move started.
```

Comment:
I don't. I didn't even realise you would want the attacker to retreat to the country it hopped over.
And this would be strange, as you would retreat with hopping, if the hopped-over country was friendly but not owned.

This also poses the problem of having multiple viable to-be-hopped-over countries, and where it wasn't clearly declared in advance (because irrelevant for the attack).